common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -70,6 +71,12 @@
         docClass += ' has-fixed';
     }
 
+    if (supportsSupports && testCssSupportForPropertyAndValue('position', 'sticky')) {
```

Comment:
> `&&` prevents falling into the else of `testCssSupportForPropertyAndValue`

is the idea that you won't have sticky if you don't have `supports`? 

because `testCssSupportForPropertyAndValue` already has a value at this point, it looks like you're testing for `supports` again. 

given it's a bit confusing, i can imagine someone removing that in the future. does it really save that much time? if you'd rather leave it as it stands,  do you fancy a comment explaining why the second check?